tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/conversion/MavenConversionTest.java

```diff
@@ -88,7 +88,18 @@ public void deleteDependenciesAfterConversion(){
 		PropertyDialog pd = openPropertiesProject(WEB_PROJECT_NAME);
 		new DefaultTreeItem("Java Build Path").select();
 		new DefaultTabItem("Libraries").activate();
-		List<TreeItem> it = new DefaultTree(1).getItems();
+		
+		List<TreeItem> itRootDirs = new DefaultTree(1).getItems();
```

Comment:
I think you can use RedDeer implementation of this dialog:
https://github.com/eclipse/reddeer/blob/master/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/jdt/ui/preferences/BuildPathsPropertyPage.java
WDYT?

Reply:
deleteDependenciesAfterConversion test rewritten using BuildPathsPropertyPage.